test/e2e/framework/helpers/yunikorn/yk_utils.go

```diff
@@ -21,6 +21,13 @@ package yunikorn
 import (
 	"fmt"
 
+	"github.com/apache/yunikorn-core/pkg/common/configs"
+	"github.com/apache/yunikorn-k8shim/pkg/common/constants"
+
+	//	"github.com/apache/yunikorn-k8shim/pkg/common/constants"
```

Comment:
Nit: delete comment

Reply:
Good catch. Will remove on commit if that's the only required change.